addons/website_blog/controllers/main.py

```diff
@@ -100,6 +100,10 @@ def _prepare_blog_values(self, blogs, blog=False, date_begin=False, date_end=Fal
             if use_cover and not fullwidth_cover:
                 offset += 1
 
+        if search:
+            tags_like_search = BlogTag.search([('name', 'like', search)])
+            domain += ['|', '|', '|', ('author_name', 'like', search), ('name', 'like', search), ('content', 'like', search), ('tag_ids', 'in', [tag.id for tag in tags_like_search])]
```

Comment:
Prefer the `ilike` operator instead of `like`

`[tag.id for tag in tags_like_search]` => `tags_like_search.ids`

Reply:
blog post is website specific, so need to append the `website.website_domain()` part

Reply:
And please, handle the search part in another commit, since their are no link with the tag part